src/components/MapView.tsx

```diff
@@ -333,10 +333,10 @@ export const MapView = () => {
           clickBusStop={clickBusStop}
           isConnectionMode={mapMode === MapModes.connection}
           isViewMode={mapMode === MapModes.view}
-          inReportMode={mapMode === MapModes.report}
+          isReportMode={mapMode === MapModes.report}
         />
         <NewReportMarker newReportCoordinates={newReportCoordinates} />
-        <ImportedReports reports={importedReports} />
+        <ImportedReports reports={importedReports} resetActiveStop={setActiveBusStopId} />
```

Comment:
`resetActiveStop={() => setActiveBusStopId(null)}` - działa to samo, ale funkcja z 'reset' w nazwie od razu resetuje, bez dodatkowego parametru